src/app/tickets/ticket-detail/ticket-detail.component.html

```diff
@@ -106,8 +106,8 @@ <h4>Opis</h4>
     <div class="button-div">
       <button class="btn btn-primary edit-button" (click)="editTicket()" *ngIf="isViewMode">Edytuj</button>
       <button class="btn btn-info edit-button w-10" (click)="cancel()" *ngIf="!isViewMode">Odrzuć</button>
-      <button class="btn btn-primary edit-button w-10" (click)="update()" *ngIf="!isViewMode">Zapisz</button>
-
+      <button class="btn btn-primary edit-button w-10" (click)="update()" *ngIf="!isViewMode" [disabled]="!editForm.valid">Zapisz</button>
+      <button class="btn btn-danger edit-button w-10" (click)="closeTicket(ticket.id)" *ngIf="!isViewMode" [disabled]="!editForm.valid">Zamknij</button> 
```

Comment:
Moim zdaniem powinien być guzik edit, a bok niego Zamknij, i ten Zamknij nie powinien się pojawiać dopiero po kliknięciu Edytuj - Edytuj|Zamknij. Klikasz Edytuj to masz Odrzuć|Zapisz. Dodatkowe też musi być modal, czy na pewno.